maxim/logger/components/generation.py

```
@@ -124,51 +137,96 @@ def get_generation_config_dict(
     type: Literal["audio"]
     transcript: str
 
 
 class GenerationResultMessage(TypedDict):
+    """Generation result message.
+
+    This class represents a generation result message.
+    """
+
     role: str
     content: Optional[Union[List[Union[TextContent, ImageContent, AudioContent]], str]]
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)